File: sdk/go/provider/sink_fblay_com.go
package provider

// sink.fblay.com：mailinator 官方姊妹子域名，MX 指向 mail.mailinator.com。

func SinkFblayComGenerate() (*CreatedMailbox, error) {
	local := randomStr(12)
	return &CreatedMailbox{
		Channel: "sink-fblay-com",
		Email:   local + "@sink.fblay.com",
	}, nil
}

func SinkFblayComGetEmails(email string) ([]NormEmail, error) {
	return MailinatorGetEmails(email)
}

File: sdk/python/tempmail_sdk/providers/sink_fblay_com.py
"""Mailinator 姊妹域名 sink.fblay.com"""

from typing import List
from ..types import Email, EmailInfo
from . import mailinator

CHANNEL = "sink-fblay-com"
DOMAIN = "sink.fblay.com"


def generate_email() -> EmailInfo:
    local = mailinator._random_string(12)
    return EmailInfo(channel=CHANNEL, email=f"{local}@{DOMAIN}")


def get_emails(token: str, email: str = "", **kwargs) -> List[Email]:
    return mailinator.get_emails(token, email, **kwargs)
